Tools/AP_Periph/can.cpp

```diff
@@ -90,40 +84,8 @@ static struct instance_t {
 #elif CONFIG_HAL_BOARD == HAL_BOARD_SITL
     HALSITL::CANIface* iface;
 #endif
-
-#if HAL_PERIPH_CAN_MIRROR
-  #if HAL_NUM_CAN_IFACES < 2
-    #error "Can't use HAL_PERIPH_CAN_MIRROR if there are not at least 2 HAL_NUM_CAN_IFACES"
-  #endif
-    ObjectBuffer<AP_HAL::CANFrame> *mirror_queue;
-    uint8_t mirror_fail_count;
-#endif // HAL_PERIPH_CAN_MIRROR
 } instances[HAL_NUM_CAN_IFACES];
 
-
-#if CONFIG_HAL_BOARD == HAL_BOARD_CHIBIOS && defined(HAL_GPIO_PIN_TERMCAN1) && (HAL_NUM_CAN_IFACES >= 2)
```

Comment:
have we lost the termination software support?

Reply:
No, this is a unused bit of code, the termination lines are handelled differenty now. https://github.com/ArduPilot/ardupilot/blob/1355bec80c7a2260e05524af820bec58882f6e63/Tools/AP_Periph/can.cpp#L830

Reply:
It is used by Here4AP and by HerePro, the others look OK